assume/scenario/loader_csv.py

```diff
@@ -728,7 +728,7 @@ def load_scenario_folder(
     study_case: str,
     perform_evaluation: bool = False,
     terminate_learning: bool = False,
-    episode: int = 1,
+    episode: int = 0,
     eval_episode: int = 1,
```

Comment:
should eval_episode be 0 as well?

Reply:
I didn't look into this change really, just applied it by recommendation in #512 but it did not help me reproduce an issue